app/public/js/init.js

```diff
@@ -37,7 +37,8 @@ function playsNice(cb) {
       return
     }
     this.done = 1
-    cb(!(event && (event.error || event.type === 'error' || event.lineno)))
+    cb(!(event && (false)))
+    // TODO: REVERT cb(!(event && (event.error || event.type === 'error' || event.lineno)))
```

Comment:
why we needed to comment out this?

Reply:
@xdzurman

Reply:
Can't find the original PR but I think this was a quick hack fix of trezor crashing the site during the pool owner functionality presentation.